api/src/modules/proxy/proxy.controller.ts

```diff
@@ -161,4 +161,60 @@ export class ProxyController {
   async proxyFeaturesTile(@Req() request: Request, @Res() response: Response) {
     return this.proxyService.proxyTileRequest(request, response);
   }
+
+  @ApiOperation({
+    description: 'Get planning unit geometries.',
+  })
+  /**
+   *@todo Change ApiOkResponse mvt type
+   */
+  @ApiOkResponse({
+    type: 'mvt',
+  })
+  @ApiUnauthorizedResponse()
+  @ApiForbiddenResponse()
+  @ApiParam({
+    name: 'z',
+    description: 'The zoom level ranging from 0 - 20',
+    type: Number,
+    required: true,
+  })
+  @ApiParam({
+    name: 'x',
+    description: 'The tile x offset on Mercator Projection',
+    type: Number,
+    required: true,
+  })
+  @ApiParam({
+    name: 'y',
+    description: 'The tile y offset on Mercator Projection',
+    type: Number,
+    required: true,
+  })
+  @ApiQuery({
+    name: 'bbox',
+    description: 'Bounding box of the project',
+    type: Array,
+    required: false,
+    example: [-1, 40, 1, 42],
+  })
+  @ApiQuery({
+    name: 'planningUnitGridShape',
+    description: 'Planning unit grid shape',
+    type: String,
+    required: false,
+  })
+  @ApiQuery({
+    name: 'planningUnitAreakm2',
+    description: 'Planning unit area in km2',
+    type: Number,
+    required: false,
+  })
+  @Get('/planning-units/preview/tiles/:z/:x/:y.mvt')
```

Comment:
may I suggest to start moving these endpoints to the controller they logically belong to?

In practice, we should get rid of this ProxyController, after all, I think - unless we have proxied requests that don't belong elsewhere. So this specific new endpoint in this PR could be moved to a (new in this case) `PlanningUnitsController` within the (existing) `PlanningUnitsModule`.

If you agree with moving these endpoints, feel free to do this in a separate PR, of course.

The main benefit from my point of view is that we would be grouping them together with related endpoints (so the features tiles endpoint could go to the existing GeoFeaturesController, and so on), which also helps with the generated Swagger documentation.